src/test/java/org/openlmis/requisition/service/RequisitionServiceTest.java

```diff
@@ -688,33 +688,19 @@ public void shouldFailValidationIfUserHasNoUpdateRightsAssigned() {
     when(permissionService.canUpdateRequisition(requisitionDto.getId()))
         .thenReturn(ValidationResult.noPermission("no.permission"));
 
-    ValidationResult result = requisitionService.validateCanSaveRequisition(requisitionDto,
-        requisitionDto.getId());
+    ValidationResult result = requisitionService.validateCanSaveRequisition(requisitionDto.getId());
 
     assertTrue(result.hasErrors());
     assertEquals(FailureType.NO_PERMISSION, result.getError().getType());
   }
 
   @Test
-  public void shouldFailValidationIfIdsDoNotMatch() {
```

Comment:
@sebbrudzinski But this tests just mock IdsMismatch valdation, it doesn't test code that was moved from service to controller.